packages/core/addon/components/cell-renderers/dimension.js

```diff
@@ -58,12 +72,17 @@ export default Component.extend({
   /**
    * @property {String} value - value that should be displayed in table cell
    */
-  value: computed('descField', 'idField', 'data', function() {
-    let descField = get(this, 'descField'),
+  value: computed('dimensionField', 'descField', 'idField', 'data', function() {
+    let dimensionField = get(this, 'dimensionField'),
+      descField = get(this, 'descField'),
       idField = get(this, 'idField'),
       data = get(this, 'data');
 
-    if (data[descField]) {
+    if (dimensionField) {
+      if (data[dimensionField]) {
```

Comment:
what if the dim is `=== 0`?

Reply:
how is it different from the old logic (`if (data[descField])...` or `else if (data[idField])...)`?

Reply:
how is it different from `if (data[descField])` or `if (data[idField])`?

Reply:
Can we use `isEmpty`?